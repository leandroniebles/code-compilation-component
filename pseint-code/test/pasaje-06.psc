subproceso x=lala(a)
FinSubProceso
Proceso sin_titulo
	lala(1,2)
FinProceso
