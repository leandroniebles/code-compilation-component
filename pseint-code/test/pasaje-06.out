=== Line 4: SynError 310
=== Line 4: SynError 268
